src/utils/address.cljs

```diff
@@ -72,3 +72,16 @@
           abbreviated-public-key   (str first-part-of-public-key ellipsis last-part-of-public-key)]
       abbreviated-public-key)
     nil))
+
+(defn get-short-wallet-address
+  [value]
+  (when value
+    (str (subs value 0 5) "..." (subs value (- (count value) 3) (count value)))))
+
+(defn get-network-short-name
```

Comment:
I have this as a sub data being passed in here ->
https://github.com/status-im/status-mobile/pull/17650 there's a similar approach for some of the components if you want to take a look.
quo components should just receive this data as a prop. having this as a utility in status-im with hard coded data that is then being used as a function in quo is a bit of a strange pattern imo.
Also we have in the guidelines to never use status-im2 code inside quo components as it should be treated as its own module.

Reply:
@J-Son89 I used this function https://github.com/status-im/status-mobile/pull/17736/files#diff-bc3f8bdb0dac047ed51780262bf909c8aad0cbb2d0fa8bf10fdaef3ab74be31fR40 as suggested by @ulisesmac above. 

Do you want the short name to be passed as a prop? Wouldn't it be too much repitition?

> having this as a utility in status-im 
This is not in `status-im`. I can move this function to the same file.

Reply:
double D'oh! 😓  but still I don't think we need this utility at all. I would recommend to remove this utility @ulisesmac. It feels strange to call this a utility tbh, it has hardcoded data baked in to it. I would consider the code in utils to be somewhat more generic than that.

Simply we define this data once in application code (status-im ns), and we pass it to the components to render. I don't believe it is too much repetition.